static/style.css

```diff
@@ -51,7 +51,6 @@ ul#comics-list > li {
 	margin: 0;
 	padding: 8px;
 	border-top: 1px solid #aaa;

```

Comment:
```suggestion
	border-top: 1px solid #aaa;
	background-color: #f0f0f0;
```
Without setting a background color here, we'll see "No results found".
![image](https://github.com/user-attachments/assets/8e2098f9-9c1c-41dd-9162-a99460b864f3)